pkg/server/helpers_selinux_test.go

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 package server
 
 import (
+	"fmt"
```

Comment:
Sorry. Actually I don't quite understand the test.
Could you change the desciption to "should xxx when xxx"? So that I could understand what you are willing to test.

Reply:
Will do, Thanks :)

Reply:
Done.